GUIDE.md

```diff
@@ -45,11 +45,11 @@ You can run es-node from a pre-built Docker image, a pre-built executable, or fr
 
 ## From pre-built executables
 
-Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs) and [snarkjs](#install-snarkjs).
+Before running es-node from the pre-built executables, ensure that you have installed [Node.js](#install-nodejs), [snarkjs](#install-snarkjs) and [WSL](https://learn.microsoft.com/en-us/windows/wsl/) if you are on Windows. 
```

Comment:
I would suggest removing the "Install WSL" section below and embed the installing link into the WSL directly

Reply:
The issue looks not to be resolved, so reopen it.